.gitignore

```diff
@@ -20,3 +20,4 @@ node_modules/
 /public/main.css
 .idea
 *.iml
+.clj-kondo/imports
```

Comment:
This ignore incorrect, library clj-kondo configuration should be commited to the repo.

Reply:
Ah true, also clj-kondo documentation mentions that https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#importing
> Typically, you will want to check copied configs into version control with your project.